src/main/java/net/pms/database/MediaTableTVSeries.java

```diff
@@ -180,38 +179,38 @@ private static void createTable(final Connection connection) throws SQLException
 		execute(connection,
 			"CREATE TABLE " + TABLE_NAME + "(" +
 				"ID					IDENTITY			PRIMARY KEY	, " +
-				"ENDYEAR			VARCHAR2(1024)					, " +
-				"IMDBID				VARCHAR2(1024)					, " +
+				"ENDYEAR			VARCHAR(1024)					, " +
+				"IMDBID				VARCHAR(1024)					, " +
 				"THUMBID			BIGINT							, " +
-				"PLOT				VARCHAR2(20000)					, " +
-				"STARTYEAR			VARCHAR2(1024)					, " +
-				"TITLE				VARCHAR2(1024)		NOT NULL	, " +
-				"SIMPLIFIEDTITLE    VARCHAR2(1024)		NOT NULL	, " +
-				"TOTALSEASONS		DOUBLE							, " +
-				"VERSION			VARCHAR2(1024)					, " +
-				"VOTES				VARCHAR2(1024)					, " +
-				"CREATEDBY VARCHAR2, " +
-				"CREDITS VARCHAR2, " +
-				"EXTERNALIDS VARCHAR2, " +
-				"FIRSTAIRDATE VARCHAR2, " +
-				"HOMEPAGE VARCHAR2, " +
-				"IMAGES VARCHAR2, " +
-				"INPRODUCTION BOOLEAN, " +
-				"LANGUAGES VARCHAR2, " +
-				"LASTAIRDATE VARCHAR2, " +
-				"NETWORKS VARCHAR2, " +
-				"NUMBEROFEPISODES DOUBLE, " +
-				"NUMBEROFSEASONS DOUBLE, " +
-				"ORIGINCOUNTRY VARCHAR2, " +
-				"ORIGINALLANGUAGE VARCHAR2, " +
-				"ORIGINALTITLE VARCHAR2, " +
-				"PRODUCTIONCOMPANIES VARCHAR2, " +
-				"PRODUCTIONCOUNTRIES VARCHAR2, " +
-				"SEASONS VARCHAR2, " +
-				"SERIESTYPE VARCHAR2, " +
-				"SPOKENLANGUAGES VARCHAR2, " +
-				"STATUS VARCHAR2, " +
-				"TAGLINE VARCHAR2" +
+				"PLOT				VARCHAR(20000)					, " +
+				"STARTYEAR			VARCHAR(1024)					, " +
+				"TITLE				VARCHAR(1024)		NOT NULL	, " +
+				"SIMPLIFIEDTITLE    VARCHAR(1024)		NOT NULL	, " +
+				"TOTALSEASONS		DOUBLE PRECISION				, " +
+				"VERSION			VARCHAR(1024)					, " +
+				"VOTES				VARCHAR(1024)					, " +
+				"CREATEDBY			VARCHAR, " +
+				"CREDITS			VARCHAR, " +
+				"EXTERNALIDS		VARCHAR, " +
+				"FIRSTAIRDATE		VARCHAR, " +
+				"HOMEPAGE			VARCHAR, " +
+				"IMAGES				VARCHAR, " +
+				"INPRODUCTION		BOOLEAN, " +
+				"LANGUAGES			VARCHAR, " +
+				"LASTAIRDATE		VARCHAR, " +
+				"NETWORKS			VARCHAR, " +
+				"NUMBEROFEPISODES	DOUBLE PRECISION, " +
+				"NUMBEROFSEASONS	DOUBLE PRECISION, " +
+				"ORIGINCOUNTRY		VARCHAR, " +
+				"ORIGINALLANGUAGE	VARCHAR, " +
+				"ORIGINALTITLE		VARCHAR, " +
+				"PRODUCTIONCOMPANIES VARCHAR, " +
+				"PRODUCTIONCOUNTRIES VARCHAR, " +
+				"SEASONS			VARCHAR, " +
+				"SERIESTYPE			VARCHAR, " +
+				"SPOKENLANGUAGES	VARCHAR, " +
+				"STATUS				VARCHAR, " +
+				"TAGLINE			VARCHAR" +
```

Comment:
Please use space-indentation when the indentation isn't at the start of the line, otherwise we end up with unpredictable indentation depending on IDE tab-size settings (as we can see this doesn't line up on GitHub)

Reply:
I view that most all table use tabs.
I corrected